server/db/migrate/20190623093635_create_api_users.rb

```diff
@@ -4,6 +4,7 @@ def change
 
       t.string :first_name
       t.string :last_name
+      t.string :username
```

Comment:
You shouldn't change an existing migration but create a new one.

The reason is, that since I've already executed that migration (on `master`) rails belives that it doesn't need to consider it again.. It notes down the name (actually the timestamp in a table named `schema_migrations`) and will never execute that again. Even if you change the migration as in this case.

So you would be better of adding the change to the table as a new migration (read https://guides.rubyonrails.org/active_record_migrations.html#changing-tables if you need inspiration)

Reply:
I know I should be running migrations instead of just reestablishing the whole database. So far it has been easier to reestablish it, and I have included instructions in the readme which explains how to remove and reestablish the database if it is not behaving as expected. I will start using migrations though.